profiler_measurements/channelFreezeScenario.h

```diff
@@ -0,0 +1,114 @@
+#ifndef PIPERT_CHANNELFREEZE_SCENARIO_H
+#define PIPERT_CHANNELFREEZE_SCENARIO_H
+
+
+
+#include <string>
+#include <thread>
+#include <math.h>
+
+#include "pipert/Profiler.h"
+#include "pipert/Scheduler.h"
+#include "utils.h"
+
+/// Measurement Title : drop rate
+/// in this measurement we will generate three channels one of them will
+/// have high drop rate
+///
+///
+////Expected analysis output
+/// to find the channel with the highst drop rate
+
+namespace measurements {
+    class NumberTocalculate {
+    public:
+        NumberTocalculate(int number) : number_(number){}
+        int Value() const { return number_; }
+
+    private:
+        int number_;
+    };
+
+    class NumberAbsolute {
+    public:
+        NumberAbsolute(pipert::PolledChannel<NumberTocalculate> *pc_to_write)
+                : pc_to_write_(pc_to_write) {}
+
+
+        void getNumberAbsolute(pipert::PacketToProcess<NumberTocalculate> p) {
+
+            pc_to_write_->Acquire(p.timestamp() + 10, abs(p.data().Value()) );
+        }
+
+    private:
+        pipert::PolledChannel<NumberTocalculate> *pc_to_write_;
+    };
+
+    class NumberSqrt {
+    public:
+
+
+        void freeze(){
+            int timeout=10000000;
```

Comment:
Porbably you want to put all implemention here to a cpp file as this code is currently inlined in the header.